data/migrations/24.lua

```diff
@@ -1,3 +1,7 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 25 (Guildhall support)")
+	db.query("ALTER TABLE `guilds` ADD `house_id` int(11) NOT NULL DEFAULT '0'")
```

Comment:
```suggestion
```
See my other suggested change in ioguild.cpp on how to load this with a left join instead.